tools/python/stylesheet/drules_merge.py

```diff
@@ -106,13 +106,13 @@ def apply_diff(drules, diff):
 
 if __name__ == '__main__':
   if len(sys.argv) <= 3:
-    print 'Usage: {} <drules1.bin> <drules2.bin> <drules_out.bin> [drules_out.txt]'.format(sys.argv[0])
+    print('Usage: {} <drules1.bin> <drules2.bin> <drules_out.bin> [drules_out.txt]'.format(sys.argv[0]))
     sys.exit(1)
 
   drules1 = drules_struct_pb2.ContainerProto()
-  drules1.ParseFromString(open(sys.argv[1]).read())
+  drules1.ParseFromString(open(sys.argv[1], mode='rb').read())
```

Comment:
Why binary for ParseFromString?

Reply:
`open` has different default mode in python 2 vs 3. 
In python3 by default mode to convert newlines and after this new lines conversation (bytes like `0xa`) google protobuf library can not understand bytes received from`open(sys.argv[1]).read()`, while with python2 all work fine.

Reply:
Wow, good to know, thanks!